CONTRIBUTING.md

```diff
@@ -101,7 +111,8 @@ In addition, the `README.md` should also point to Psammead's [CODE_OF_CONDUCT](h
 ### After merge
 
 After merging a pull request into `latest`, please follow these steps to ensure that the changes are published to npm and to the Psammead Storybook https://bbc-news.github.io/psammead.
+
 - Publish Packages  
-  To make your changes available on the npm registry please publish any package changes. The instructions to do this can be found in the the [repository README](https://github.com/BBC-News/psammead#rocket-publishing-packages) 
+  To make your changes available on the npm registry please publish any package changes. The instructions to do this can be found in the the [repository README](https://github.com/BBC-News/psammead#rocket-publishing-packages)
```

Comment:
I don't think this was an intended change of your PR, but is either of these sections ("publish packages", "publish storybook") still necessary? CI handles this for us & we document that process elsewhere.

Reply:
Agreed, we should remove - let's raise another PR for that